ardcvocabs/src/main/java/au/org/aodn/ardcvocabs/service/ArdcVocabServiceImpl.java

```diff
@@ -31,6 +33,7 @@ public class ArdcVocabServiceImpl implements ArdcVocabService {
 
     public ArdcVocabServiceImpl(RestTemplate restTemplate) {
         this.restTemplate = restTemplate;
+        this.retryTemplate = new RetryTemplate();
```

Comment:
Ok, you should do it like this

1. You already create the retryTemplate using @Bean
2. The ArdcVocabServiceImpl is create in the config so all you need to do it add the parameter in the creation so it autowrite the RetryTemplate and then you can set it via the constructore